addons/website_livechat/static/src/components/discuss/discuss.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-inherit="mail.Discuss.content" t-inherit-mode="extension">
-        <xpath expr="//*[hasclass('o_Discuss_thread')]" position="before">
-            <t t-if="discuss.thread.visitor">
+    <t t-inherit="mail.ThreadView" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o_ThreadView_threadViewTopbar')]" position="after">
+            <t t-if="threadView.thread.visitor">
```

Comment:
Extension is changed from component `Discuss` to `ThreadView`.
It should not longer be in `components/discuss/discuss.xml` but instead `components/thread_view/thread_view.xml`.

It's fine not to do in stable, but it should be done in master.